maps/upload.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<title>Planetary Response Network: file uploader</title>
+<link rel="stylesheet" type="text/css" href="/css/main.css">
+<main id="upload-page">
+  <form>
```

Comment:
The API endpoint hasn't been set up yet but I'd add `method=post` and `action=https://maps-api.planetaryresponsenetwork.org/uploads` to stub this out.